### DISCUSSION TITLE
Hide the "Pin" column

### DISCUSSION THREAD

**Discussion by TobiasSenger** [KevinVandy/material-react-table#1288] [Q&A] [answered]
Hi everyone, I need to hide the "Pin" column that is automatically shown whenever I set "enablePinning: true". I do require the pinning logic but I do not want to show the separate "Pin" column. I would be very happy if anyone could help me out with this. Thank you! Best, Tobi

**Comment by KevinVandy** [comment, marked as answer]
https://www.material-react-table.com/docs/guides/display-columns
